### DISCUSSION TITLE
How do I configure Rectangle Pro to always position an app in a specific location when opened?

### DISCUSSION THREAD

**Discussion by ShawnCBerg** [rxhanson/RectanglePro-Community#515] [Q&A]
I have configured App Layouts and they work well if I execute them manually, but I really want specific apps to always open in a specific location. Likewise, I'd also always like certain apps to open stashed to the right side of my screen. Is any of this possible? Been a longtime Rectangle open source user and giving the Pro version a shot.

**Comment by rxhanson** [comment]
Thanks for trying Pro! I’ve avoided adding this to the app because it would require constantly polling macOS to check the existing windows and see when a new one is opened. I might make this a hidden feature for users that are ok with the app using more resources to achieve this.
